---
layout: default
title: Research
---

## Research

I aim to contribute to minimizing the suffering caused by cancer by developing computational methods that extract actionable information from tumors. 

<a href="https://doi.org/10.3929/ethz-b-000708983"><img style="float: right;" src="../assets/img/thesis.png" width=250></a>
Describing the heterogeneous populations of cells that constitute tumors is an important step towards designing effective cancer therapies. With the development of single-cell sequencing technologies, this heterogeneity can be accessed at very high resolution and used to understand tumor progression. I develop statistical models and learning algorithms to describe the heterogeneity of tumors from high-throughput single-cell sequencing data. During my PhD, I designed and implemented three computational tools for single-cell RNA and DNA sequencing data analysis, and applied these methods to discover subgroups within a large cohort of metastatic melanoma patients. My doctoral thesis is available <a href="https://doi.org/9.3929/ethz-b-000708983">here</a> and the cover features a beautiful drawing by <a href="https://ivoliveira.com">In&ecirc;s Viegas Oliveira</a>.

I am enthusiastic about sharing my research in conferences and meetings. You can find a list of some of my talks and poster presentations below.

I do my best to provide constructive criticism to authors of papers submitted to journals and conferences. I reviewed submissions to Nature Communications, Cell Reports Methods, Genome Research, Genome Biology, Bioinformatics, BMC Bioinformatics, PLoS Computational Biology, RECOMB and ISMB.

<br>

### Selected papers
Please see my [Google Scholar page](https://scholar.google.com/citations?hl=en&user=wBVuepgAAAAJ&view_op=list_works&sortby=pubdate) for a complete list.
* Jack Kuipers * , Mustafa A. Tuncel * , Pedro F. Ferreira * , Katharina Jahn, Niko Beerenwinkel. Single-cell copy number calling and event history reconstruction. 2025.
* Pedro F. Ferreira, Jack Kuipers, Niko Beerenwinkel. Identifying hierarchical cell states and gene signatures with deep exponential families for single-cell transcriptomics. 2024.
* Pedro F. Ferreira, Jack Kuipers, Niko Beerenwinkel. Mapping single-cell transcriptomes to copy number evolutionary trees. 2021.

<br>

### Talks, lectures and posters
* EMBO workshop in Spatial Biology of Cancer, London, UK. Hierarchical cell states in single-cell and spatial transcriptomics data from tumors. (poster)
* FARO Seminar in Oncology, Basel, CH. Tumor single-cell genomic and transcriptomic data analysis and integration with applications to melanoma.
* ICSDS 2023 session on "Omics data analysis", Lisbon, PT. Deep exponential families for single-cell data analysis. (invited talk)
* Basel Computational Biology Conference (BC2), Basel, CH. Deep exponential families for single-cell data. (poster)
* OLISSIPO workshop, INESC-ID, Lisbon, PT. Analysis of single-cell data from tumors. (workshop)
* D-BSSE internal seminar, Basel, CH. Understanding tumor evolution through single-cell data.
* Raphael group, Princeton University, NJ, USA. Understanding tumor heterogeneity through single-cell data. 
* Statistics seminar, UBC, Vancouver, CA. Understanding tumor heterogeneity through single-cell data. 
* CytoData Symposium, Allen Institute for Cell Science, Seattle, WA, USA. Understanding tumor heterogeneity through single-cell data. (invited talk)
* RECOMB 2022, San Diego, CA, USA. Mapping single-cell transcriptomes to copy number evolutionary trees. (contributed talk)
* OLISSIPO exchange, Lisbon, Portugal. Mapping single-cell transcriptomes to copy number evolutionary trees. 
* Ascona workshop, Ascona, CH. Mapping single-cell transcriptomes to copy number evolutionary trees.
* Cancer evolution. Bad Honnef, DE. Mapping single-cell transcriptomes to copy number evolutionary trees. (poster)
* D-BSSE VMB seminar, Basel, CH. Matrix factorization in single-cell data.
* SIB PhD student retreat, Davos, CH. Mapping single-cell transcriptomes to copy number evolutionary trees. 
