---
layout: default
title: Research
---

## Research

I aim to contribute to minimizing the suffering caused by cancer by developing computational methods that extract actionable information from tumors. During my PhD, I have been creating methods to identify hierarchical gene signatures from gene expression data, reconstruct the history of copy number aberrations in cancer cells, and associate genomic with transcriptomic states from single-cell sequencing data. You can find a list of selected papers below. 

I am enthusiastic about sharing my research in conferences and meetings. You can find a list of some of my talks and poster presentations below.

I do my best to provide constructive criticism to authors of papers submitted to journals and conferences. I have been fortunate to be able to review submissions to Nature Communications, Cell Reports Methods, Genome Research, Genome Biology, Bioinformatics, BMC Bioinformatics, PLoS Computational Biology, RECOMB and ISMB.

<br>

### Selected papers
Please see my [Google Scholar page](https://scholar.google.com/citations?hl=en&user=wBVuepgAAAAJ&view_op=list_works&sortby=pubdate) for a complete list.
* Pedro F. Ferreira, Jack Kuipers, Niko Beerenwinkel. Deep exponential families for single-cell data analysis. 2022.
* Pedro F. Ferreira, Jack Kuipers, Niko Beerenwinkel. Mapping single-cell transcriptomes to copy number evolutionary trees. 2021.
* Jack Kuipers * , Mustafa A. Tuncel * , Pedro F. Ferreira * , Katharina Jahn, Niko Beerenwinkel. Single-cell copy number calling and event history reconstruction. 2020.

<br>

### Talks, lectures and posters
* OLISSIPO workshop, INESC-ID, Lisbon, PT. Analysis of single-cell data from tumors. (workshop)
* D-BSSE internal seminar, Basel, CH. Understanding tumor evolution through single-cell data.
* Raphael group, Princeton University, NJ, USA. Understanding tumor heterogeneity through single-cell data. 
* Statistics seminar, UBC, Vancouver, CA. Understanding tumor heterogeneity through single-cell data. 
* CytoData Symposium, Allen Institute for Cell Science, Seattle, WA, USA. Understanding tumor heterogeneity through single-cell data. (Invited talk)
* RECOMB 2022, San Diego, CA, USA. Mapping single-cell transcriptomes to copy number evolutionary trees. (Contributed talk)
* OLISSIPO exchange, Lisbon, Portugal. Mapping single-cell transcriptomes to copy number evolutionary trees. 
* Ascona workshop, Ascona, CH. Mapping single-cell transcriptomes to copy number evolutionary trees.
* Cancer evolution. Bad Honnef, DE. Mapping single-cell transcriptomes to copy number evolutionary trees. (poster)
* D-BSSE VMB seminar, Basel, CH. Matrix factorization in single-cell data.
* SIB PhD student retreat, Davos, CH. Mapping single-cell transcriptomes to copy number evolutionary trees. 
